keyboards/wilba_tech/wt65_a/keymaps/via/rules.mk

```diff
@@ -1,71 +1,3 @@
 # project specific files
```

Comment:
I'm thinking you maybe meant to delete this line?

Reply:
thanks